docs/introduction/start-building-your-application.md

```diff
@@ -113,6 +113,24 @@ switch ($domainId) {
     case 3: $defaultCurrency = $this->getReference(CurrencyDataFixture::CURRENCY_CZK); break;
     ...
 ```
+
+## Fine-tune your configuration
+If all developers working on your project use the same version of PHP (eg. because all use SSFW via Docker), you can use higher versions of the libraries and tools installed via Composer.
+To do so, remove the `config.platform.php` option from your `composer.json`:
```

Comment:
Or specify a newer version?

Reply:
That wouldn't have any real advantage for teams that use the same PHP version. Or do you see any?

Reply:
Will not it be actuall when php 7.4 is out?

Reply:
It might, if the one of the developers from the team starts using it and others don't.